x-pack/legacy/plugins/maps/public/layers/sources/client_file_source/create_client_file_source_editor.js

```diff
@@ -6,7 +6,7 @@
 
 
 import React from 'react';
-import { JsonUploadAndParse } from '../../../../../file_upload/public';
+import { start as fileUpload } from '../../../../../file_upload/public/legacy';
```

Comment:
see above. if we're going to rename it here, just give it it's usable name when exporting in legacy

Reply:
I definitely see your point, but I stuck with prevailing conventions on this one. If you do a local code search for "start as" you'll see some NP examples.